login-workflow/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@brightlayer-ui/react-auth-workflow",
-    "version": "4.0.0",
+    "version": "4.0.1",
```

Comment:
We need to update the package version once we fix all issues which need to be part of this patch release.

Reply:
We update the package version in dev branch in the first PR related to that version. Here's an example we did it in the past https://github.com/etn-ccis/blui-react-workflows/pull/169/files